terraform/modules/prom-ec2/prometheus/targets.tf

```diff
@@ -0,0 +1,61 @@
+resource "aws_s3_bucket" "prometheus_targets" {
+  bucket        = "govukobserve-targets-${var.environment}"
```

Comment:
Insert comment about how we don't have `Observe` as a concept anymore 😄

Reply:
yeah i know but naming is hard and we already use this prefix for the config bucket, so consistency or something 😄